client/js/api-reference/client-methods.mdx

```diff
@@ -38,37 +32,84 @@ This method initiates the connection process, optionally passing parameters that
     </Expandable>
 </ParamField>
 
+During the startBot process, the transport state will transition through the states: "authenticating" and "authenticated".
+
+```javascript
+try {
+  await pcClient.startBot({
+    endpoint: "/api/start", // Your server endpoint to start the bot
+    requestData: {
+      initial_prompt: "You are a pirate captain",
+      llm_provider: "openai"
+    }
+  });
+} catch (error) {
+  console.error("Error starting the bot:", error);
+}
+```
+
+### connect()
+
+`async connect(connectParams): Promise<unknown>`
+
+This method initiates the connection process, optionally passing parameters that your transport class requires to establish a connection or an endpoint to your server for obtaining those parameters.
```

Comment:
I thought we no longer wanted to advertise this method as taking "an endpoint to your server for obtaining those parameters"